dev/tidb-in-kubernetes/deploy/gcp-gke.md

```diff
@@ -550,8 +584,8 @@ terraform destroy
 
 管理多个 Kubernetes 集群的最佳实践有以下两点：
 
-1. 为每个 Kubernetes 集群创建一个新目录；
-2. 根据具体需求，使用 Terraform 脚本将上述模块进行组合。
+- 为每个 Kubernetes 集群创建一个新目录；
```

Comment:
Suggest to keep the original format, we have to create the dir first and then organize the modules in the dir.

Reply:
Got it! Will update.